docs/spark/how-to-guides/ubuntu-instructions.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Building .NET for Apache Sapark application on Ubuntu
+description: Learn how to build your .NET for Apache Spark application on Ubuntu
+ms.date: 01/02/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+
+# Table of Contents
+- [Open Issues](#open-issues)
+- [Pre-requisites](#pre-requisites)
+- [Building](#building)
+  - [Building Spark .NET Scala Extensions Layer](#building-spark-net-scala-extensions-layer)
+  - [Building .NET Sample Applications using .NET Core CLI](#building-net-sample-applications-using-net-core-cli)
+- [Run Samples](#run-samples)
+
+## Open Issues:
+- [Building through Visual Studio Code]()
+
+## Pre-requisites:
+
+If you already have all the pre-requisites, skip to the [build](ubuntu-instructions.md#building) steps below.
+
+  1. Download and install **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** or the **[.NET Core 3.0 preview SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0)** - installing the SDK will add the `dotnet` toolchain to your path.
```

Comment:
.NET Core 3.0 is no longer preview AFAIK.

```suggestion
  1. Download and install **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** or the **[.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0)** - installing the SDK will add the `dotnet` toolchain to your path.
```

Reply:
```suggestion
  1. Download and install **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** or the **[.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0)** - installing the SDK adds the `dotnet` toolchain to your path.
```